### PR TITLE
User info jwt

### PR DESCRIPTION
**Pull request by DanielElisenberg**
## User info from JWT to database

Userinfo from the JWT token in the cookies header are required for starting a new job, and have to contain information about the user starting the job. Most test structure and supporting functions lifted from admin-backend

### CODE REVIEW

File: .test.env [hunk 1]
```diff
@@ -1,4 +1,6 @@
 MONGODB_URL=mongodb://localhost:27017/jobdb
-SECRETS_FILE=test/resources/secrets/secrets.json
-INPUT_DIR=test/resources/input_directory
-DOCKER_HOST_NAME=localhost
\ No newline at end of file
+JWKS_URL=http://jwks.test
+SECRETS_FILE=tests/resources/secrets/secrets.json
+INPUT_DIR=tests/resources/input_directory
+DOCKER_HOST_NAME=localhost
+STACK=prod
```
Comment (pawbu) [inline .test.env:6]: Why STACK is set to prod in a `.test.env `file?

File: job_service/api/auth.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+def get_jwks_aud() -> str:
+    return 'datastore-qa' if environment.get('STACK') == 'qa' else 'datastore'
+
+
+def get_signing_key(jwt_token: str):
+    return jwks_client.get_signing_key_from_jwt(jwt_token).key
+
+
+def authorize_user(token: Union[str, None]) -> UserInfo:
```
Comment (pawbu) [inline job_service/api/auth.py:28]: Maybe we should also check the `accreditation/role` here? Or should anyone be able to add new jobs?

File: tests/unit/model/test_job.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 from job_service.model.job import Job
 
 
-RESOURCE_DIR = 'test/resources/model'
+RESOURCE_DIR = 'tests/resources/model'
```
Comment (pawbu) [inline tests/unit/model/test_job.py:8]: Good catch 👍

File: job_service/api/auth.py (hunk 2 above)
Comment (DanielElisenberg) [reply job_service/api/auth.py:28]: That is a good idea! I am on it 🏃🏻

**Review by pawbu — APPROVED** [review]
💯

**Pull request merged**